src/agentNet/agent-net.py

```diff
@@ -0,0 +1,37 @@
+import networkx as nx
+import random
+import matplotlib.pyplot as plt
+import enum
+
+class DensityProb(enum.Enum):
+        SPARSE = 0.1
+        MEDIUM = 0.5
+        DENSE = 0.8
+def dens2prob(dens_name):
+            return DensityProb[dens_name].value
+
+class AgentNet(object):
+    """
+    Class with agent network graph.
+    """
+    def __init__(self, agent_num=None, net_type=None, net_density=None):
+        self.agent_num =  agent_num if agent_num is not None else 2
```

Comment:
co to jest agent_num?

Reply:
`agent_num` odpowiada za liczbę agentów, którzy będą tworzyć sieć połączeń.

Reply:
jeżeli tak to dlaczego metody typu add_agent albo load_yaml nie modyfikują tego?
lepiej by chyba było zrobić to (jak już musi być) jako read-only property zwracające liczbę węzłów w self.network

Reply:
Nazwa zmiennej jest nieco myląca -- w założeniu to ma być inicjalna liczba agentów, tj. początkowa liczba wierzchołków w grafie losowym (parametr algorytmu), a nie licznik agentów w sieci w ogóle.